CHANGELOG.md

```diff
@@ -34,6 +34,9 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## Unreleased
 
+## [v0.1.4](https://github.com/provenance-io/provenance/releases/tag/v0.1.4) - 2021-02-16
```

Comment:
I believe the v0.1.4 tag should into place on an RC branch instead of here so we can merge in other PRs without merge conflicts...

Reply:
That's confusing.  So PR to main, then branch for release, tag there, then merge back to main?

Reply:
just going off the contributing.md file that lists all those steps ... and yeah you pull off main to start that process